build/commands/lib/util.js

```diff
@@ -406,6 +406,14 @@ const util = {
     fs.copySync(srcDir, dstDir)
   },
 
+  // Add mojom ast patcher.
+  addMojomAstPatcher: () => {
```

Comment:
add this to python path instead in config.js

Reply:
moved patcher to `chromium_src/python_modules`